icons/heart-handshake.svg

```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M20.42 4.58a5.4 5.4 0 0 0-7.65 0l-.77.78-.77-.78a5.4 5.4 0 0 0-7.65 0C1.46 6.7 1.33 10.28 4 13l8 8 8-8c2.67-2.72 2.54-6.3.42-8.42z" />
-  <path d="M12 5.36 8.87 8.5a2.13 2.13 0 0 0 0 3h0a2.13 2.13 0 0 0 3 0l2.26-2.21a3 3 0 0 1 4.22 0l2.4 2.4" />
-  <path d="m18 15-2-2" />
-  <path d="m15 18-2-2" />
+  <path d="M19.1,14.1L12,21l-7.1-6.9C3.4,12.6,2,10.8,2,8.5C2,5.5,4.5,3,7.5,3C9.3,3,11,3.9,12,5.3 c1.7-2.5,5.2-3.1,7.7-1.3C21.1,5,22,6.7,22,8.5C22,10.8,20.6,12.6,19.1,14.1z" />
+  <path d="M12,5.4L8.9,8.5c-0.8,0.8-0.8,2.2,0,3l0,0c0.8,0.8,2.2,0.8,3,0l2.3-2.2c1.2-1.2,3.1-1.2,4.2,0l2.6,2.6" />
+  <path d="M18.1,15.1L16,13" />
+  <path d="M15,18l-2-2" />
```

Comment:
Yeah, the base heart shape I suggested is not going to work, since this icon needs the two halves to meet at a 90deg angle exactly on the diagonal, so:
```suggestion
  <path d="M19 14c1.49-1.46 3-3.21 3-5.5A5.5 5.5 0 0 0 16.5 3c-1.76 0-3 .5-4.5 2-1.5-1.5-2.74-2-4.5-2A5.5 5.5 0 0 0 2 8.5c0 2.3 1.5 4.05 3 5.5l7 7" />
  <path d="M12 5 9.04 7.96a2.17 2.17 0 0 0 0 3.08v0c.82.82 2.13.85 3 .07l2.07-1.9a2.82 2.82 0 0 1 3.79 0l2.96 2.66" />
  <path d="m18 15-2-2" />
  <path d="m15 18-2-2" />
```